formatTest/typeCheckedTests/expected_output/arityConversion.re

```diff
@@ -16,11 +16,11 @@ module Test = {
     | Or (int, int);
 };
 
-Test.And (1, 2);
+let _ = Test.And (1, 2);
```

Comment:
The loss we currently have for 4.02.3 is when converting from ocaml to reason. I'd be curious to have you confirm if the loss actually still occurs.